spec/take_away_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'take_away'
+
+describe TakeAway do
+
+  subject(:take_away) { described_class.new }
+
+  describe 'class' do
+    it 'creates an instance of a class' do
+      expect(take_away).to be_a(TakeAway)
+    end
+  end
+
+  describe '#read_menu' do
+    it 'prints menu upon request' do
+      expect { take_away.read_menu }.to output.to_stdout
+    end
+  end
+
+  describe '#add_to_order' do
+    it 'stores added items to order_hash' do
+      expect {take_away.add_to_order(1,1)}.to change{take_away.order_hash.length}.by(1)
+    end
+    it 'prints items that were added to the order' do
+      expect { take_away.read_menu }.to output.to_stdout
+    end
+  end
+
+  describe '#place_order' do
+    let(:order_hash) { double :order_hash }
+    before do
+      allow(take_away).to receive(:finalize_order)
+    end
+    it { is_expected.to respond_to(:place_order).with(1).argument }
+    it 'saves an order' do
+      # Figure out how to test!
+      # (take_away.order).stub(:total) { "total" }
+      # expect(customer.order).to eq("total")
+    end
```

Comment:
Indeed, I have moved some take_away functionality to order, and it seems to work and be tested better, and look neater!